jcl/src/java.base/share/classes/openj9/internal/foreign/abi/InternalUpcallHandler.java

```diff
@@ -56,6 +60,19 @@ public final class InternalUpcallHandler {
 	private UpcallMHMetaData metaData;
 
 	/*[IF JAVA_SPEC_VERSION >= 19]*/
+	/*[IF JAVA_SPEC_VERSION >= 20]*/
+	/**
+	 * The constructor creates an upcall handler specific to the requested java method
+	 * by generating a native thunk in upcall on a given platform.
+	 *
+	 * @param target The target method handle in upcall
+	 * @param mt The method type of the target method handle
+	 * @param cDesc The function descriptor of the target method handle
+	 * @param session The segment scope related to the upcall handler
+	 * @return an internal upcall handler with the thunk address
+	 */
+	public InternalUpcallHandler(MethodHandle target, MethodType mt, FunctionDescriptor cDesc, SegmentScope session)
+	/*[ELSE] JAVA_SPEC_VERSION >= 20 */
```

Comment:
Please restructure things like this to use `ELSEIF`, so instead of this pattern:
```
	/*[IF JAVA_SPEC_VERSION >= 19]*/
	/*[IF JAVA_SPEC_VERSION >= 20]*/
	/* java 20 code */
	/*[ELSE] JAVA_SPEC_VERSION >= 20 */
	/* java 19 code */
	/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
	/*[ELSE] JAVA_SPEC_VERSION >= 19 */
	/* java 16-18 code */
	/*[ENDIF] JAVA_SPEC_VERSION >= 19 */
```
use just one if/.../endif sequence:
```
	/*[IF JAVA_SPEC_VERSION >= 20]*/
	/* java 20 code */
	/*[ELSEIF JAVA_SPEC_VERSION >= 19]*/
	/* java 19 code */
	/*[ELSE] JAVA_SPEC_VERSION >= 19 */
	/* java 16-18 code */
	/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
```

Reply:
Updated at all places involved (I think `/*[ELSEIF JAVA_SPEC_VERSION == 19]*/` is more appropriate in such case).